### ISSUE TITLE
 FutureWarning: load_stylesheet() will not receive pyside parameter in version 3

### ISSUE THREAD

**Issue by vfxpandit** [ColinDuquesnoy/QDarkStyleSheet#125]
I am Using Python3.6 and PySide2
This theme is working fine in my PySide2 app.
But I am receive this Warning:

```
C:\Python36\lib\site-packages\qdarkstyle\__init__.py:214: FutureWarning: load_stylesheet() will not receive pyside parameter in version 3. Set QtPy environment variable to specify the Qt binding insteady.
  FutureWarning

```

**Comment by dpizetta** [comment]
What is the QdarkStyle version you are using? Tks for reporting.

**Comment by flutefreak7** [comment]
I think I've always seen that warning on recent versions. Has that changed
recently? Are we not supposed to see that anymore?

**Comment by dpizetta** [comment]
Sorry I've not seen in the title version 3. I also corrected that in v2.6.6. I'll change to deprecation warning, so, users should have not seen this.

Some more info about it: https://docs.python.org/3/library/warnings.html
